app/controllers/dashboards_controller.rb

```diff
@@ -15,4 +15,12 @@ def show
     @feed = Dashboard.display_feed
   end
 
+  def send_flow
+    flow = Flowdock::Flow.new(:api_token => ENV['LP_FLOW'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.